lib/text-formatters.js

```diff
@@ -1,27 +1,30 @@
 /**
  * Commonly-used functions for formatting text in badge labels. Includes
- * ordinal numbers, currency codes, star ratings, etc.
+ * ordinal numbers, star ratings, etc.
  */
+
 'use strict';
 
+const assert = require('assert');
+
+function versionString(version) {
+  const prefix = /^[0-9]/.test(version) ? 'v' : '';
+  return prefix + version;
+}
+exports.versionString = versionString;
+
 function starRating(rating) {
-  var stars = '';
-  while (stars.length < rating) { stars += '★'; }
-  while (stars.length < 5) { stars += '☆'; }
-  return stars;
+  assert(rating >= 0);
```

Comment:
Also note that rounding changed from `Math.ceil` (implicit) to `Math.round` (explicit), so `starRating(1.01)` gives ★☆☆☆☆ instead of ★★☆☆☆.